ale/drivers/viking_drivers.py

```diff
@@ -0,0 +1,61 @@
+import spiceypy as spice
+
+import ale
+from ale.base.data_naif import NaifSpice
+from ale.base.label_isis import IsisLabel
+from ale.base.type_sensor import Framer
+from ale.base.base import Driver
+
+class VikingIsisLabelNaifSpiceDriver(Framer, IsisLabel, NaifSpice, Driver):
+
+    @property
+    def spacecraft_name(self):
```

Comment:
You should add an instrument_id property similar to the other drivers that maps the label names to the NAIF names. This will help load(s) run faster for other drivers.

Reply:
Then, you can re-use that here.